misaligned.js

```diff
@@ -1,16 +1,35 @@
 const {expect} = require('chai')
+ const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
+ const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
+let colorSet= new Map();
 
 function print_color_map() {
-    const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
-    const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
-    for (let i = 0; i < majorColors.length; i++) {
+   for (let i = 0; i < majorColors.length; i++) {
         for (let j = 0; j < minorColors.length; j++) {
             console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
         }
     }
     return majorColors.length * minorColors.length;
 }
 
+function printColorMapStub() {
+     for (let i = 0; i < majorColors.length; i++) {
+        for (let j = 0; j < minorColors.length; j++) {
+    colorSet.set((i * 5 + j)+1,{'majorColor':majorColors[i],'minorColor':minorColors[j]});
+          }
+        }
+}
+function checkColorMap(colorCode,expectMajorColor,expectedMinorColor){
+  printColorMapStub();

```

Comment:
The stub is tested, but it does not test the real functionality. Even if `print_color_map` has a bug, this test will not find it.